app/database/models_v2.py

```diff
@@ -0,0 +1,310 @@
+    )
+    email = Column(
+        String,
+        unique=True,
+        nullable=False,
+        doc="A unique email. Email must be valid.",
+    )
+    full_name = Column(
+        String,
+        nullable=False,
+        doc="The user's full name. A full name must be at least 2 characters"
+        " long.",
+    )
+    is_active = Column(
+        Boolean,
+        default=True,
+        nullable=False,
+        doc="Whether the user account is active or temporarily disabled."
+        " Defaults to True.",
+    )
+    is_admin = Column(
+        Boolean,
+        default=False,
+        nullable=False,
+        doc="Whether the user is a site admin. Defaults to False.",
+    )
+    language_id = Column(
+        Integer,
+        ForeignKey("language.id"),
+        default=1,
+        nullable=False,
+        doc="The user's UI language's ID. Defaults to 1.",
+    )
+    password = Column(
+        String,
+        nullable=False,
+        doc="The user's password. A password must be a minimum length of 3"
+        " characters and a maximum length of 20 characters.",
+    )
+    privacy = Column(
+        Enum(PrivacyKinds),
+        default=PrivacyKinds.Private,
+        nullable=False,
+        doc="The user's privacy setting. Defaults to PrivacyKinds.Private.",
+    )
+    target_weight = Column(
+        Float,
+        nullable=True,
+        doc="The user's target weight goal.",
+    )
+    telegram_id = Column(
+        String,
+        unique=True,
+        nullable=True,
+        doc="A unique Telegram ID.",
+    )
+    username = Column(
+        String,
+        unique=True,
+        nullable=False,
+        doc="A unique username. A valid username must be a minimum length of 3"
+        " characters, a maximum length of 20 characters, and use only the"
+        " following characters: a-zA-Z0-9_.",
+    )
+    events = relationship(
+        "Event",
+        secondary="user_event",
+        cascade="all, delete",
+        back_populates="members",
+        doc="A list of events the user is attending.",
+    )
+    language = relationship(
+        "Language",
+        doc="The user's UI language.",
+    )
+    owned_events = relationship(
+        "Event",
+        cascade="all, delete",
+        back_populates="owner",
+        doc="A list of events the user created.",
+    )
+
+    def __repr__(self):
+        return f"<User {self.id}>"
+
+
+user_event = Table(
+    "user_event",
+    Base.metadata,
+    Column("user.id", Integer, ForeignKey("user.id")),
+    Column("event.id", Integer, ForeignKey("event.id")),
+)
+
+
+class Event(Base):
+    """A database model of an Event entity."""
+
+    __tablename__ = "event"
+
+    id = Column(
+        Integer,
+        primary_key=True,
+        index=True,
+        nullable=False,
+        doc=PRIMARY_KEY_DOC,
+    )
+    # category_id = Column(
+    #     Integer,
+    #     ForeignKey("categories.id"),
+    #     nullable=False,
+    #     doc="",  # TODO: doc
+    # )
```

Comment:
Should we add it or not? :)

Reply:
I haven't done all classes, but probably better to remove commented out and re-add it later.

Reply:
Done.